### PR TITLE
Shadowling fixes

### PR DESCRIPTION
**Pull request by danhibiki**
Fixes weird visual artifacts that come at variable light levels in certain situations as a shadowling/thrall.  Also fixes phasing to work when ascended. Credits to @grufus2 for pointing out where the errors were.

### CODE REVIEW

File: html/changelogs/Dan Hibiki - lingfixes.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a changelog editing system that should cause fewer conflicts and more accurate timestamps."
+  - rscdel: "Killed innocent kittens."
```
Comment (ghost) [inline html/changelogs/Dan Hibiki - lingfixes.yml]: you forgot to edit the changelog

**Comment by Spacedong** [comment]
lol rip

**Comment by ghost** [comment]
:+1:

**Comment by danhibiki** [comment]
Oh woops, had the changes in notepad but forgot to save

**Comment by Nokington** [comment]
sure

**Pull request merged**